examples/secure-third-party-request/package.json

```diff
@@ -11,7 +11,7 @@
         "typecheck": "tsc --noEmit"
     },
     "dependencies": {
-        "@frontify/app-bridge-app": "^0.0.16",
+		"@frontify/app-bridge-app": "^0.1.3",
```

Comment:
Some rogue indentation still

Reply:
thx for noticing, i pushed a reformat for that...